cross/fontconfig/PLIST

```diff
@@ -0,0 +1,58 @@
+bin:bin/fc-cache
+bin:bin/fc-cat
+bin:bin/fc-list
+bin:bin/fc-match
+bin:bin/fc-pattern
+bin:bin/fc-query
+bin:bin/fc-scan
+bin:bin/fc-validate
+etc/fonts/conf.d/10-scale-bitmap-fonts.conf
+etc/fonts/conf.d/20-unhint-small-vera.conf
+etc/fonts/conf.d/30-metric-aliases.conf
+etc/fonts/conf.d/30-urw-aliases.conf
+etc/fonts/conf.d/40-nonlatin.conf
+etc/fonts/conf.d/45-latin.conf
+etc/fonts/conf.d/49-sansserif.conf
+etc/fonts/conf.d/50-user.conf
+etc/fonts/conf.d/51-local.conf
+etc/fonts/conf.d/60-latin.conf
+etc/fonts/conf.d/65-fonts-persian.conf
+etc/fonts/conf.d/65-nonlatin.conf
+etc/fonts/conf.d/69-unifont.conf
+etc/fonts/conf.d/80-delicious.conf
+etc/fonts/conf.d/90-synthetic.conf
+etc/fonts/conf.d/README
+etc/fonts/fonts.conf
+lnk:lib/libfontconfig.so
+lnk:lib/libfontconfig.so.1
+lib:lib/libfontconfig.so.1.8.0
+share/fontconfig/conf.avail/10-autohint.conf
```

Comment:
- Missing prefixes in this file
- Set `rsc:share/fontconfig/conf.avail/` to keep the size of the file down. Some other dirs here can use a similar treatment.